### PR TITLE
Add test_token_is_allowed()

### PR DESCRIPTION
**Pull request by sevamove**
Add `test_token_is_allowed()` in order to test the function `tokenIsAllowed()` from `contracts/TokenFarm.sol`

### CODE REVIEW

**Comment by PatrickAlphaC** [comment]
This is a good PR! Thanks!

We just need to make that function a view function to return a bool.

**Comment by sevamove** [comment]
Thanks for letting me know how to improve this PR! #3

File: tests/unit/test_token_farm.py [hunk 1]
```diff
@@ -25,6 +25,17 @@ def test_add_allowed_tokens():
...
+    # Arrange.
+    if network.show_active() not in LOCAL_BLOCKCHAIN_ENVIRONMENTS:
+        pytest.skip("Only for local testing!")
+    account = get_account()
+    token_farm, dapp_token = deploy_token_farm_and_dapp_token()
+    # Act.
+    test_add_allowed_tokens()
+    token_is_allowed = token_farm.tokenIsAllowed(dapp_token, {"from": account})
+    # Assert.
+    assert token_is_allowed == True
```
Comment (PatrickAlphaC) [inline tests/unit/test_token_farm.py]:
```suggestion
    assert token_is_allowed is True
```

**Review by PatrickAlphaC — CHANGES_REQUESTED** [review]
Just one change :)

**Comment by PatrickAlphaC** [comment]
Thanks for this PR!!

**Pull request merged**